packages/okta-angular/README.md

```diff
@@ -144,6 +171,52 @@ const oktaConfig = {
 };
 ```
 
+## Reference
+### `oktaAuth`
+`oktaAuth` is the top-most component of `okta-angular`. This is where most of the configuration is provided.
+
+#### Configuration Options
+  - `issuer` **(required)**: The OpenID Connect `issuer`
+  - `clientId` **(required)**: The OpenID Connect `client_id`
+  - `redirectUri` **(required)**: Where the callback is hosted
+  - `scope` *(optional)*: Reserved or custom claims to be returned in the tokens
+  - `onAuthRequired` *(optional)*: Accepts a callback to make a decision when authentication is required. If not supplied, `okta-angular` will redirect directly to Okta for authentication.
+
+#### `oktaAuth.loginRedirect(uri, additionalParams?)`
+Performs a full page redirect to Okta based on the initial configuration.  On successful authentication, the router will navigate to the path specified with the `uri` parameter. 
+
+If you have an Okta `sessionToken`, you can bypass the full-page redirect by passing in this token. This is recommended when using the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget). Simply pass in a `sessionToken` into the `loginRedirect` method follows:
+
+```typescript
+// Navigate to "/profile" on login success
+this.oktaAuth.loginRedirect('/profile', {
+  sessionToken: /* sessionToken */
+})
+```
+
+> Note: For information on obtaining a `sessionToken` using the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget), please see the [`renderEl()` example](https://github.com/okta/okta-signin-widget#rendereloptions-success-error).
```

Comment:
I know we're trying to wrap Auth JS and eventually take away the direct access, but we should put some kind of documentation in here about that these "additional params" are - even if it's just linking to the AuthJS method we're delegating to: https://github.com/okta/okta-auth-js#tokengetwithredirectoptions